Source/items.cpp

```diff
@@ -4013,7 +4016,8 @@ void PrintItemDetails(const Item &item)
 		AddPanelString(PrintItemPower(item._iPrePower, item));
 	}
 	if (item._iSufPower != -1) {
-		AddPanelString(PrintItemPower(item._iSufPower, item));
+		if ((item._itype == ItemType::Bow && gbIsHellfire) || (item._iFlags & ISPL_QUICKATTACK) == 0)

```

Comment:
Why are you only including the suffix power in the panel string under these specific cases?

Reply:
So it works as intended

Reply:
Fair enough - I read something backwards so it wasn't making sense to me. Why not just blank out quick attack under PrintItemPower, since you are doing all that logic there anyway? Then return the string properly formatted for whatever version of the game you are using.

Reply:
Because we need to prevent a blank line from being added

Reply:
This one probably could have stayed closed